pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -1208,6 +1226,9 @@ impl Timeline {
             let mut guard = self.layers.write().await;
             guard.finish_gc_compaction(&layer_selection, &compact_to, &self.metrics)
         };
+
+        self.remote_client
+            .schedule_compaction_update(&layer_selection, &compact_to)?;
```

Comment:
This also seems like a drive-by bug fix. How did you discover it?

Reply:
By looking at the logs there are a bunch of warnings

Reply:
...but I forgot the exact message